app/src/webview/java/org/mozilla/focus/webview/FocusWebViewClient.java

```diff
@@ -252,7 +253,7 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
     @Override
     public void onReceivedSslError(WebView view, SslErrorHandler handler, SslError error) {
         handler.cancel();
-
+        recordSSLErrors(error);
```

Comment:
Since there are two states here, we probably want to differentiate between the two.